Library/Formula/racer.rb

```diff
@@ -0,0 +1,47 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+  head "https://github.com/phildawes/racer.git"
+
+  option "without-src", "Do not get rust source for racer from www.rust-lang.org"
+  depends_on "rust" => :build
+
+  resource "rustsrc" do
```

Comment:
What do you think about naming this rust-1.3? This is so that you can have rust-1.4 when it is (soon) released.

If you change this to rust-1.3, it would make more sense to install the source into ```#{opt_prefix}/src/rust-1.3```.